src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.cs

```diff
@@ -1865,19 +1865,27 @@ public async Task PostAsync_CallMethod_EmptyContent(Configuration.Http.RemoteSer
             }
         }
 
+        public static IEnumerable<object[]> ExpectContinueVersion()
+        {
+            var versions = new string[] {"1.0", "1.1", "2.0"};
+            var expectContinue = new bool?[] {true, false, null};
+            return
+                from expect in expectContinue
+                from version in versions
+                select new object[] {expect, version};
+        }
```

Comment:
Nit: this could be a tad more concise, e.g.
```C#
public static IEnumerable<object[]> ExpectContinueVersion() =>
    from expect in new bool?[] {true, false, null}
    from version in new string[] {"1.0", "1.1", "2.0"}
    select new object[] {expect, version};
```

Reply:
Fixed